feat: new introductory javascript project (#54305)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-pyramid-generator/660f455b044d3230ed971e98.md

---
id: 660f455b044d3230ed971e98
title: Step 76
challengeType: 0
dashedName: step-76
---

# --description--

> A <dfn>`while`</dfn> loop will run over and over again until the `condition` specified is no longer true. They have the following syntax:

Reviewer: ```suggestion
A <dfn>`while`</dfn> loop will run over and over again until the `condition` specified is no longer true. It has the following syntax:
```
